src/Query.tsx

```diff
@@ -172,7 +174,7 @@ export default class Query<TData = any, TVariables = OperationVariables> extends
   fetchData(): Promise<ApolloQueryResult<any>> | boolean {
     if (this.props.skip) return false;
     // pull off react options
-    const { children, ssr, displayName, skip, client, ...opts } = this.props;
+    const { children, ssr, displayName, skip, client, disablePartialRefetch, ...opts } = this.props;
```

Comment:
I wonder, what is the reason to deconstruct this variable in here when you don't use it later anywhere.

Reply:
@dajvido To remove it from `opts`: See something like https://codeburst.io/use-es2015-object-rest-operator-to-omit-properties-38a3ecffe90